.github/workflows/rust.yml

```diff
@@ -164,4 +164,4 @@ jobs:
       run: unzip cargo-binstall-${{ matrix.target }}.zip
 
     - name: "Run binstall"
-      run: ./${{ matrix.output }} cargo-binstall --manifest-path . --no-confirm
+      run: ./${{ matrix.output }} cargo-binstall --no-confirm
```

Comment:
Let's have both for now so we test both paths

Reply:
I am actually thinking about testing about other binaries as well.

Reply:
Done!

I added more bins to test and merge job `test` into `build` so that we don't have to wait for all build workflows for different target to finish before start testing.

This also means that if testing fails, then we would not upload any artifact.